src/Symfony/Component/Lock/Store/CombinedStore.php

```diff
@@ -92,8 +93,12 @@ public function save(Key $key)
         throw new LockConflictedException();
     }
 
+    /**
+     * {@inheritdoc}
+     */
     public function waitAndSave(Key $key)
     {
+        @trigger_error(sprintf('%s::%s has been deprecated since Symfony 4.4 and will be removed in Symfony 5.0.', \get_class($this), __METHOD__), E_USER_DEPRECATED);
```

Comment:
broken notice, `__METHOD__` gives `FQCN::method`. Parentheses are missing also, and we usually don't add the `and will be removed ...` part.

Reply:
do you mean the paretheses in %s() ?